Content.Shared/Hands/EntitySystems/SharedHandsSystem.Pickup.cs

```diff
@@ -214,6 +216,11 @@ public void PickupOrDrop(
             // TODO make this check upwards for any container, and parent to that.
             // Currently this just checks the direct parent, so items can still teleport through containers.
             ContainerSystem.AttachParentToContainerOrGrid((entity, Transform(entity)));
+
+            if (dropNear && TryComp<TransformComponent>(uid, out var transformComponent))
+            {
+                TransformSystem.PlaceNextTo((entity, Transform(entity)), ((EntityUid)uid, transformComponent));
+            }
```

Comment:
```suggestion
            if (dropNear)
                TransformSystem.PlaceNextTo(entity, uid.Value);
```
No need to do a `TryComp` for the `TransformComponent`, since every entity has it.
Also the `Resolve` inside the `PlaceNextTo` function gets it for you if you don't pass it.
Also wouldn't this fail in the case that `uid` is `null`? You are just casting it from a nullable to an `EntityUid`, but that will only remove the IDE warning which is there for a purpose.

Reply:
Didn't know they all had transform, neat. 

As for the null issue, TryComp accepts a nullable uid and will return false if it doesn't have a value (if I'm reading it correctly), so it shouldn't be possible to reach the PlaceNextTo with uid being null.

I'll replace the TryComp with a normal null check; I had it there because I thought it would be a good way to a) verify the uid wasn't null, b) check the entity had a transform component, and c) get the transform component if it existed so it didn't need to be found again later.

Also, is using uid.Value generally better than casting when switching to a non-nullable type? A google search gives me some results about easier maintenance if the type gets changed later; are there other benefits I should know?

Reply:
`uid.Value` is more readable and clearly states the intent in my opinion. Also the IDE gives you a warning if you are using it where it might be null.
Not sure how the compiler handles these in detail, but it's probably smart enough that there is no difference in performance.